autogpt_platform/frontend/.npmrc

```diff
@@ -0,0 +1,5 @@
+# Configure pnpm to save exact versions
+save-exact=true
+
+# Disable hoisting for better dependency isolation
+hoist=false
```

Comment:
We need to pin the dependencies so we avoid mismatch also in minor versions ...  This should help with it.